src/renderer/coremods/contextMenu/plaintextPatches.ts

```diff
@@ -0,0 +1,23 @@
+import type { PlaintextPatch } from "src/types";
+
+export default [
+  {
+    find: "♫ (つ｡◕‿‿◕｡)つ ♪",
```

Comment:
This property is not required. I think your query is specific enough that we don't have to worry about this.
```suggestion
```

Reply:
Keeping a `find` eases the load on searching for that regex. it should be fine